mooncake-common/include/default_config.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include <string>
+#include <cstdint>
+#include <unordered_map>
+#include <yaml-cpp/node/node.h>
+#include <jsoncpp/json/json.h>
+
+namespace mooncake {
+class DefaultConfig {
+public:
+    struct Node {
+        YAML::Node yaml_node_;
+        Json::Value json_value_;
+    };
+
+    enum ConfigType {
+        YAML = 1,
+        JSON = 2,
+        UNKnown = 3,
+    };
+public:
+    void Load();
+    /**
+     * @brief GetInt32 retrieves an integer value from the configuration
+     * @param key The key to look up in the configuration
+     * @param val Pointer to store the retrieved value
+     * @param default_value Default value to return if the key is not found
+     * @note If the key is not found, default_value will be assigned to val
+     */
+    void GetInt32(const std::string& key, int32_t *val, int32_t default_value = 0);
```

Comment:
It would be better if we could add `const` to the Get methods.

Reply:
fix